CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(zip)
 enable_language(C)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
+set(ZIP_VERSION "0.1.9")
```

Comment:
why 0.1.9?
the latest version is `v0.1.14`, so it can go to the candidate `v0.1.15`

Reply:
I took version from appveyor.yml.

Reply:
Changed to 0.1.15.

Reply:
ok, so please push changes

Reply:
Done.